frontend/src/app/(protected)/experiments/add/components/cmabs/addCMABContext.tsx

```diff
@@ -0,0 +1,300 @@
+                      <Label className="basis-1/4 mt-3 font-medium">Name</Label>
+                      <div className="basis-3/4 flex flex-col">
+                        <Input
+                          name={`context-${index + 1}-name`}
+                          placeholder="Give the context a searchable name"
+                          value={context.name || ""}
+                          onChange={(e) => {
+                            const newContexts = [...contexts];
+                            newContexts[index].name = e.target.value;
+                            typeSafeSetExperimentState(
+                                newContexts as NewContext[]
+                              );
+                          }}
+                        />
+                        {errors[index]?.name ? (
+                          <p className="text-red-500 text-xs mt-1">
+                            {errors[index].name}
+                          </p>
+                        ) : (
+                          <p className="text-red-500 text-xs mt-1">&nbsp;</p>
+                        )}
+                      </div>
+                    </div>
+                  </Field>
+                  <Field className="flex flex-col">
+                    <div className="flex flex-row">
+                      <Label className="basis-1/4 mt-3 font-medium">
+                        Description
+                      </Label>
+                      <div className="basis-3/4 flex flex-col">
+                        <Textarea
+                          name={`context-${index + 1}-description`}
+                          placeholder="Describe the context"
+                          value={context.description || ""}
+                          onChange={(e) => {
+                            const newContexts = [...contexts];
+                            newContexts[index].description = e.target.value;
+                            typeSafeSetExperimentState(
+                                newContexts as NewContext[]
+                              );
+                          }}
+                        />
+                        {errors[index]?.description ? (
+                          <p className="text-red-500 text-xs mt-1">
+                            {errors[index].description}
+                          </p>
+                        ) : (
+                          <p className="text-red-500 text-xs mt-1">&nbsp;</p>
+                        )}
+                      </div>
+                    </div>
+                  </Field>
+                </div>
+
+                <div className="basis-1/2 grow">
+                <Field className="flex flex-col mb-4">
+                    <div className="flex flex-row">
+                    <Label className="basis-1/4 mt-3 font-medium" htmlFor="mu">
+                        Context type
+                    </Label>
+                    <div className="basis-3/4 flex flex-col">
+                    <RadioGroup
```

Comment:
Shading looks odd in light mode. Mind fixing?

<img width="872" alt="image" src="https://github.com/user-attachments/assets/18d370f4-18c4-4f9f-b8a0-6eb30a25263f" />